ogr/services/pagure/service.py

```diff
@@ -288,3 +288,12 @@ def project_create(
         except PagureAPIException as ex:
             self.__handle_project_create_fail(ex, namespace)
         return PagureProject(repo=repo, namespace=namespace, service=self)
+
+    def list_projects(
+        self,
+        namespace: str = None,
+        user: str = None,
+        search_pattern: str = None,
+        language: str = None,
+    ) -> List[GitProject]:
+        raise NotImplementedError
```

Comment:
as mentioned above, `ogr/exceptions.py` should have exception class for not supported operations :)